app/models/rules_of_origin/origin_reference_document.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module RulesOfOrigin
+  class OriginReferenceDocument
+    include ActiveModel::Model
+
+    attr_accessor :ord_title, :ord_version, :ord_date, :ord_original
+    attr_writer :id
+
+    def id
+      @id = 'origin_reference_document_id'
```

Comment:
I suspect you want to use the ContentAddressableId module and specify the fields that make this entity unique